docs/developers.md

```diff
@@ -4,6 +4,32 @@
 
 See [interface.py], which defines the interface of a backend and may serve as a template for creating new backends.
 
+### Developing the sparqlwrapper
+
+Tripper comes with an inbuilt backend to the SPARQLWrapper. In order
+to test this properly a real triplestore is needed. This is not done in the
+automatic workflows on github. However, a lcal graphDB can be setup as described below and tested with test_sparqlwrapper_graphdb.py.
+
+To create the local instance of graphdb:
+```bash
+docker pull ontotext/graphdb:10.8.3 # latest tag 17.02.2025
+docker run -d -p 7200:7200 --name graphdb ontotext/graphdb:10.8.3
+```
+
+To create a repository in the graphdb isntance:
+```bash
+curl -X POST \
+     -H "Content-Type: application/rdf+xml" \
+     --data-binary @tests/backends/graphdb_config.xml \
+     http://localhost:7200/rest/repositories
+```
+
+Then go to `http://localhost:7200/` in your browser.
+You can add a new repository by pressing `create new reposotory`in the bottom right corner. Choose `GraphDB Reposotory` choos `test_repo` a id.
+Tick off `Enable full-text search` and leave the rest as predefined.
```

Comment:
```suggestion
Then go to  [http://localhost:7200/](http://localhost:7200/) in your browser.
You can add a new repository by pressing `create new reposotory` in the bottom right corner. Choose `GraphDB Reposotory` choos `test_repo` a id.
Tick off `Enable full-text search` and leave the rest as predefined.
```

Reply:
Is it possible to create a GraphDB docker image with the needed settings turned on by default? It is probably just a configuration file that needs to be edited somewhere. That could open for starting the docker image in a GitHub workflow and include the GraphDB backend test in the CI.